src/sentry/event_manager.py

```diff
@@ -2455,6 +2459,52 @@ def _save_aggregate_performance(jobs: Sequence[PerformanceJob], projects: Projec
                 EventPerformanceProblem(event, performance_problems_by_hash[problem_hash]).save()
 
 
+@metrics.wraps("save_event.send_occurrence_to_platform")
+def _send_occurrence_to_platform(jobs: Sequence[Job], projects: ProjectsMapping) -> None:
+    for job in jobs:
+        event = job["event"]
+        project = event.project
+        event_id = event.event_id
+
+        use_platform = options.get("performance.issues.send_to_issues_platform", False)
+        if use_platform:
+            per_project_option = project.get_option(
+                "sentry:performance_issue_send_to_issues_platform", False
+            )
+            if per_project_option:
+                performance_problems = job["performance_problems"]
+                for problem in performance_problems:
+                    # handle only N+1 db query detector first
+                    if problem.type.type_id == PerformanceNPlusOneGroupType.type_id:
+                        occurrence = IssueOccurrence(
+                            id=uuid.uuid4().hex,
+                            resource_id=None,
+                            project_id=project.id,
+                            event_id=event_id,
+                            fingerprint=[problem.fingerprint],
+                            type=problem.type,
+                            issue_title=problem.title,
+                            subtitle=event.transaction,
+                            # TODO need to set metadata.value="problem.desc"
```

Comment:
Looks like I might have missed this as part of our ingestion schema. What does `metadata["value"]` get used for?

Reply:
it's used in the group stream. we generally show a query there 
<img width="886" alt="image" src="https://user-images.githubusercontent.com/23648387/225129764-92c6ec08-7729-4360-8569-17dd13dd7d6b.png">

Reply:
it's all the `SELECT`s and `GET /api/` in the screenshot

Reply:
Ok, looks like I missed this into the `IssueOccurrence` model. Right now we have `title` - `N+1 Api Call` and `subtitle` - `GET /api/...`. But we don't have anything for location, which is currently extracted from the event.

This all happens here: https://github.com/getsentry/sentry/blob/b134ebb0d4cbe3d5a03cd633d4bbb1c29cbd8545/src/sentry/issues/ingest.py#L121-L139

Feels to me like `subtitle` should map to `metadata["value"]`, and we should have a new `location` value that maps to `metadata["location"]`, and we fall back to `event["location"]` if not present. Does that make sense to you?

Reply:
this is currently where the values are coming from in the UI 
<img width="629" alt="Screenshot 2023-03-14 at 5 53 53 PM" src="https://user-images.githubusercontent.com/23648387/225149172-015bbd99-bc7d-4908-8ac5-00507b8d1659.png">

for title, if there's no title in metadata, we'd default to pulling title from the group which is fine
for culprit, we read it from the group

so we don't really use location in this specific case

Reply:
it seems that in the UI we don't really use `subtitle` that's sent https://github.com/getsentry/sentry/blob/master/static/app/utils/events.tsx#L112

Reply:
> Feels to me like subtitle should map to metadata["value"], and we should have a new location value that maps to metadata["location"], and we fall back to event["location"] if not present. Does that make sense to you?

I think this can work. we're flipping culprit and metadata.value since the section right of the title is called a subtitle and the string below is called message but it should be fine I think

Reply:
Ahh interesting. It looks like `culprit` just happens to be the same as `metadata["location"]` for perf issues, so I made that assumption.

Ok, so I think that we should just make `culprit` explicit as part of the occurrence then. So
 - `title` - `metadata["title"]`
 - `subtitle` - `metadata["value"]`
 - `culprit` - `culprit`

Reply:
yeah there are some fields that I don't think we use ever. 
👍 that works! I updated subtitle and can follow up with more changes after this PR

Reply:
https://github.com/getsentry/sentry/pull/45809